### PR TITLE
Avoid notifications

### PR DESCRIPTION
**Pull request by oliviertassinari**
We were 6 to receive this notification today: https://groups.google.com/a/mui.com/g/job/c/eEtcZkS7pvk. It would have been better if only the person that is hiring for the role got it. I have also updated https://app.ashbyhq.com/admin/organizational-settings/email-templates/bced3350-b4cb-488f-b35f-c4157e6f489b accordingly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -130,5 +130,5 @@ The goal of this second phase is to improve the component from v5.0.0-alpha.15 a
 ## Submission
 
 **DO NOT** host your project on a public repository.
-Please send us a zip file containing this project (with the _.git_ but without the _node_modules_ folders) using the upload link that we have provided by email (or send it at job@mui.com).
+Please send us a zip file containing this project, with the _.git_ but without the _node_modules_ folders; using the upload link that we have provided by email. If you don't have the upload link, you can simply send it at job@mui.com.
```
Comment (oliviertassinari) [inline README.md:133]:
```suggestion
Please send us a zip file containing this project, with the _.git_ but without the _node_modules_ folders, using the upload link that we have provided by email. If you don't have the upload link, you can simply send it to job@mui.com.
```

File: README.md [hunk 2]
```diff
@@ -130,5 +130,5 @@ The goal of this second phase is to improve the component from v5.0.0-alpha.15 a
 ## Submission
 
 **DO NOT** host your project on a public repository.
-Please send us a zip file containing this project (with the _.git_ but without the _node_modules_ folders) using the upload link that we have provided by email (or send it at job@mui.com).
+Please send us a zip file containing this project, with the _.git_ but without the _node_modules_ folders, using the upload link that we have provided by email. If you don't have the upload link, you can simply send it to job@mui.com.
```
Comment (oliviertassinari) [inline README.md:133]:
```suggestion
Please send us a zip file containing this project, with the _.git_ but without the _node_modules_ folders, using the upload link that we have provided by email.
If you don't have the upload link, you can simply send it to job@mui.com.
```

**Review by mnajdova — APPROVED** [review]
Sounds good.

**Comment by oliviertassinari** [comment]
Hopefully, fewer notifications for everyone 🙌

**Pull request merged**